storage/innobase/xtrabackup/src/xbcrypt.c

```diff
@@ -97,12 +90,26 @@ static struct my_option my_long_options[] =
 	 &opt_encrypt_chunk_size, &opt_encrypt_chunk_size, 0,
 	GET_ULL, REQUIRED_ARG, (1 << 16), 1024, ULONGLONG_MAX, 0, 0, 0},
 
+	{"encrypt-threads", 't',
+	 "Number of threads for parallel data encryption/decryption. "
+	 "The default value is 1",
+	 &opt_encrypt_threads, &opt_encrypt_threads, 0,
+	 GET_UINT, OPT_ARG, 1, 1, UINT_MAX, 0, 0, 0},
+
 	{"verbose", 'v', "Display verbose status output.",
 	 &opt_verbose, &opt_verbose,
 	  0, GET_BOOL, NO_ARG, 0, 0, 0, 0, 0, 0},
 	{0, 0, 0, 0, 0, 0, GET_NO_ARG, NO_ARG, 0, 0, 0, 0, 0, 0}
 };
 
+/* Need the following definitions to avoid linking with ds_*.o and their link
```

Comment:
Nope, in fact we build few files of datasink as source files for the xbcrypt, the same way it is used in xbstream.